packages/core/database/src/metadata/index.ts

```diff
@@ -36,46 +36,26 @@ export const createMetadata = (models: Model[] = []): Metadata => {
 
   // init pass
   for (const model of _.cloneDeep(models)) {
-    if ('id' in model.attributes) {
-      throw new Error('The attribute "id" is reserved and cannot be used in a model');
-    }
```

Comment:
should we prevent this somewhere else? (pbbly in the transform-content-types-to-models utils) There is a reserved attribute names in there

Reply:
Sure I can readd the check for the CTs 👍

Reply:
Update done :)